_blog-src/_posts/2015-06-15-apiman-msa-day.adoc

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title:  "Microservices Architecture Day Appearance"
+date:   2015-06-15 17:30:00 +0100
+author: marc_savy
+categories: microservices conference talk
+---
+
+I had the pleasure of presenting on apiman at the recent [Microservices Architecture Developer Day], with our Fabric8 friend Kurt delivering a short demo of apiman in Fabric8. It was particularly enjoyable meeting developers who are interested in, or are already using, apiman - so, thank you for your insightful questions both during, and after, the presentation.
+
+Given the packed schedule, there was a limited amount of time to explore apiman plus microservices, and hence this seems like a good opportunity to write a blog post expanding upon the themes I touched upon.
+
+So, if you're interested in understanding the value the API management can have in a microservices architectures; please, read on!
```

Comment:
Thanks, rewording screw up. Ended with half of the old sentence, half of the new. Fixed.